contracts/near/eth-client/src/tests.rs

```diff
@@ -525,13 +599,15 @@ fn add_two_blocks_from_400000() {
 
     let mut contract = EthClient::init(
         true,
+        String::from("ethash"),
         400_000 / 30000,
         vec![blocks_with_proofs.first().unwrap().merkle_root],
         blocks[0].clone(),
         30,
         10,
         10,
         None,
+        97,
```

Comment:
Let's use named ChainId here, to be clear what this number actually means. In this case since we are using `ethash` let's use chain_id 3, though it is not used anywhere so it makes no real difference.

Just as a note for other reviewing this PR I found a list of chain id from networks supporting EVM: [Chainlist](https://chainlist.org/).